backends/alpm/pk-alpm-transaction.c

```diff
@@ -194,6 +194,34 @@ pk_alpm_transaction_progress_cb (alpm_progress_t type, const gchar *target,
 	g_assert (pkalpm_current_job);
 	job = pkalpm_current_job;
 
+	if (target == "") {
```

Comment:
does that work? Surely you mean NULL? Otherwise it probably needs to be `g_strcmp0`

Reply:
The string isn't a null pointer in this case, but a `""` string.

Reply:
Then you'll have to change it to `if (g_strcmp0(target, "") == 0) {` as @hughsie  said.